### ISSUE TITLE
Earnings Bugs

### ISSUE THREAD

**Issue by Tarek-Bohdima** [Greenstand/treetracker-admin#626]
Earnings page:

- “Sub organization name” column is missing
- Payment date column has no values or any feedback like "NA" or "No payment yet"

**Comment by dadiorchen** [comment]
No.1 Yea, I was thinking to add sub org name to the list, but considering the UX changing is bug risky, I didn't do that, if we consider this UX is valid, we can put it into milestone 2, I think this is a better choice. 
For now, if we want to check the sub org of a earnings, we can click the list item, and it's shown on the details panel

**Comment by dadiorchen** [comment]
This is also a UX problem, my opinion is: for any new UX suggestion, change, we should put it to the next stage, to avoid new bugs. 
Also, for discussing convenience, I split this to two issue.
